package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "2.1.8"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "devDependencies": {
+    "eslint": "^5.3.0",
```

Comment:
My system is complaining that this version of eslint is a bit ancient. It requires `5.12.0`. Any reason you chose to go with this? Or was it the default?

Reply:
After fixing my local settings, I realized npm was still giving me warnings. We are having two different versions of `eslint` installed on the same project. I don't know if everything is fine on your side, but things are not working well on mine with both. I will remove the lower version.

![Screenshot 2019-04-16 at 17 19 27](https://user-images.githubusercontent.com/19375569/56217448-f0d88680-606b-11e9-90a9-0eea60dd5211.png)